Assets/Plugins/Source/Editor/Platforms/Android/AndroidBuilder.cs

```diff
@@ -346,7 +346,7 @@ private static void ConfigureGradleTemplateProperties()
 
         private static void ConfigureEOSDependentLibrary()
         {
-            string clientIDAsLower = Config.Get<EOSConfig>().clientID.ToLower();
+            string clientIDAsLower = Config.Get<PlatformConfig>().clientCredentials.ClientId.ToLower();
```

Comment:
This should be changed to `PlatformManager.GetPlatformConfig()` instead of this line.

Reply:
Fixed. Marking conversation as resolved.